models.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <title>Drawing with Russians, Models </title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="info middle">
+        <h3>Models</h3>
+        <div class="question">I am posing at the next Drawing with Russians? What do I need to know?</div>
+        <div class="answer">Be responsive, communicate promptly with the organizer. Arrive ahead of schedule. No model has flaked since the beginning of this event, you do not want to be the first.</div>
+
+        <div class="question">How is the session structured?</div>
+        <div class="answer">The organizer will explain the house rules (no photos, add money to the jar, no talking). Timing is as follows:
+            [START]
+            5 x 1-minute poses
+            3 x 5-minute poses
+            [BREAK]
+            2 x 5-minute poses
+            1 x 10-minute pose
+            [BREAK]
+            2 x 10-minute poses
+            [BREAK]
+            1 x 20-minute pose
+            [BREAK]
+            1 x 20-minute pose
```

Comment:
We no longer have a second 20 minute pose because people were getting too anxious.